src/Resources/contao/languages/en/tl_module.php

```diff
@@ -43,3 +44,4 @@
 $GLOBALS['TL_LANG']['merger2']['inherit_all_articles']          = 'Article from parent pages';
 $GLOBALS['TL_LANG']['merger2']['inherit_articles_fallback']     = 'Article of this site <em>or</em> from parent page';
 $GLOBALS['TL_LANG']['merger2']['inherit_all_articles_fallback'] = 'Article of this site <em>or</em> from parent pages';
+$GLOBALS['TL_LANG']['merger2']['legend_module']                 = 'Modules: ';
```

Comment:
I prefer using a placeholder for the Module label here, see comment above. So the value should be `Module: %s`. To translators may change the order of the label/value.

Reply:
changed